client/stylesheets/shared/_reset.scss

```diff
@@ -14,7 +14,7 @@
 
 	@include breakpoint( '>782px' ) {
 		#wpbody-content {
-			padding: 0;
+			padding: 0 0 65px 0;
```

Comment:
common.css from WP Core has `padding-bottom: 65px;`, so we can eliminate this block altogether.

Reply:
common.css also has a `padding-left: 20px` that the original CSS was overriding. https://github.com/woocommerce/woocommerce-admin/pull/2753/commits/c45e3bc64af273cc56659de23923702dc0df40db updates the PR to only override the left padding.

Reply:
Oh, good call. I didn't see that ✅ 

LGTM then